finance-admin-api/db/invoice_adjustments.go

```diff
@@ -0,0 +1,92 @@
+package db
+
+import (
+	"github.com/ministryofjustice/opg-sirius-supervision-finance-admin/shared"
+	"os"
+	"time"
+)
+
+type InvoiceAdjustments struct {
+	FromDate *shared.Date
+	ToDate   *shared.Date
+}
+
+const InvoiceAdjustmentsQuery = `SELECT CONCAT(p.firstname, ' ', p.surname)                               AS "Customer Name",
+       p.caserecnumber                                                   AS "Customer number",
+       fc.sop_number                                                     AS "SOP number",
+       '="0470"'                                                            AS "Entity",
+       cc.code                                                           AS "Revenue cost centre",
+       cc.cost_centre_description                                        AS "Revenue cost centre description",
+       a.code                                                            AS "Revenue account code",
+       a.account_code_description                                        AS "Revenue account code description",
+       CONCAT(tt.fee_type, i.reference)                                  AS "Txn number",
+       tt.description                                                    AS "Txn description",
+       CONCAT(EXTRACT(YEAR FROM AGE(fr.enddate, fr.startdate)), ' year') AS "Remission/Exemption award term",
+       CASE
+           WHEN la.datetime >= DATE_TRUNC('year', la.datetime) + INTERVAL '3 months'
+               THEN CONCAT(TO_CHAR(la.datetime, 'YY'), '/', TO_CHAR((la.datetime + INTERVAL '1 YEAR'), 'YY'))
+           ELSE CONCAT(TO_CHAR((la.datetime - INTERVAL '1 YEAR'), 'YY'), '/', TO_CHAR(la.datetime, 'YY'))
+           END                                                           AS "Financial Year",
+       la.datetime                                                       AS "Approved date",
+       (la.amount / 100.0)::NUMERIC(10, 2)::VARCHAR(255)                               AS "Adjustment amount",
+       COALESCE(ia.notes, fr.notes)                                      AS "Reason for adjustment"
+FROM supervision_finance.ledger_allocation la
+         JOIN supervision_finance.ledger l on l.id = la.ledger_id
+         JOIN supervision_finance.invoice i on i.id = la.invoice_id
+         JOIN supervision_finance.finance_client fc on fc.id = l.finance_client_id
+         JOIN public.persons p ON fc.client_id = p.id
+         LEFT JOIN supervision_finance.invoice_adjustment ia on i.id = ia.invoice_id
+         LEFT JOIN supervision_finance.fee_reduction fr ON fr.id = l.fee_reduction_id
+         LEFT JOIN LATERAL (
+    SELECT ifr.supervisionlevel AS supervision_level
```

Comment:
This needs to change to the following to include guardianships
`SELECT CASE
            WHEN i.feetype = 'AD' THEN 'AD'
            ELSE COALESCE(sl.supervision_level, '')
            END AS supervision_level`